docker/docker-centos7-systemd-sshd/Dockerfile

```diff
@@ -2,7 +2,11 @@ FROM centos/systemd
 
 MAINTAINER "Seth Rosetter" <seth.rosetter@gmail.com>
```

Comment:
Well, it is associated with my personal Docker Hub so I don't think it is a huge deal.